### PR TITLE
 Add Test to Check World and PROTO file Headers.

### PR DESCRIPTION
**Pull request by DavidMansolino**
Add Test to Check World and PROTO file headers.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
test me

**Comment by DavidMansolino** [comment]
:+1:

File: tests/sources/test_header_version.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import os
+import fnmatch
+
+ignoredProtos = [
+    "projects/robots/mobsya/thymio/controllers/thymio2_aseba/aseba/clients/studio/plugins/ThymioVPL/UsageProfile.proto",
+    "projects/samples/tutorials/protos/FourWheelsRobot.proto"
+]
+
+
+class TestIcons(unittest.TestCase):
```
Comment (stefaniapedrazzi) [inline tests/sources/test_header_version.py]: it seems that the test class name and description have not been updated after copy/paste

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**